manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.7/MuhammadFareezIqmal.IIUMSchedule.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 0.1.0.7
+PackageLocale: en-US
+Publisher: Muhammad Fareez Iqmal
+PackageName: IIUM Schedule

```

```suggestion
PackageName: IIUM Schedule (Preview)
```
This must match Control Panel.